docs/developers-guide/substreams-sink-files.md

```diff
@@ -0,0 +1,275 @@
+2. Run code and verify output
+3. Code walkthrough
+4. Understanding the sink tool
+5. Run tool and verify output
+6. Conclusion and review
+7. Getting help
+
+## Download code and tools
+
+### Install `substreams-sink-files`
+
+**TODO:** <i>Where is the actual binary file? What is the file type/format/extension (if any)?</i>
+https://github.com/streamingfast/substreams-sink-files/releases
+
+### Verify installation
+
+Checking the version of `substreams-sink-files` will produce a message similar to:
+
+```bash
+substreams-sink-files version dev
+```
+
+### Update system `PATH`
+
+If you want to use the `substreams-sink-tool` from anywhere on your system make sure the computer's `PATH` environment variable has been set correctly.
+
+Add the following lines to the computer's `~/.bashrc` configuration file and then restart the shell session.
+
+```bash
+export GOPATH=$HOME/go
+export PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+```
+
+### Download accompanying code example
+
+The accompanying code example is available in its [official GitHub repository](#). You'll need to download or clone the repository to your computer to complete this tutorial.
+
+You can use the `git clone` command to acquire the example code.
+
+```bash
+git clone TODO_INSERT_REPO_LINK_WHEN_AVAILABLE
+```
+
+## Run code and verify output
+
+It's a good idea to run the example code using your installation of the `substreams` CLI to make sure everything is set up and working properly.
+
+Verify the setup for the example project by using the `make build`, `make codegen` and `substreams run` commands.
+
+### Build the example project
+
+First, build the Substreams module by using the included `make` command.
+
+```bash
+make
+```
+
+### Generate the required code for the example
+
+Next, run the included `make codegen` command to create the required protobuf files.
+
+```bash
+make codegen
+```
+
+### Run the example
+
+Last, run the project by using the `substreams run` command.
+
+```bash
+substreams run -e mainnet.eth.streamingfast.io:443 substreams-filesink-tutorial.yaml map_eth_block_for_sink --start-block 1000000 --stop-block +1
+```
+
+### Results of example
+
+The `substreams run` command will result in output resembling the following:
+
+```bash
+----------- NEW BLOCK #1,000,000 (1000000) ---------------
+{
+  "@module": "map_eth_block_for_sink",
+  "@block": 1000000,
+  "@type": "substreams.sink.files.v1.Lines",
+  "@data": {
+    "lines": [
+      "{\"hash\":\"8e38b4dbf6b11fcc3b9dee84fb7986e29ca0a02cecd8977c161ff7333329681e\",\"number\":1000000,\"parent_hash\":\"b4fbadf8ea452b139718e2700dc1135cfc81145031c84b7ab27cd710394f7b38\",\"timestamp\":\"2016-02-13T22:54:13Z\"}"
+    ]
+  }
+}
+```
+
+## Code walkthrough
+
+Take a moment to review the various files in the accompanying Substreams project. Two important files to review include the [Substreams manifest](#) and the [`lib.rs`](#) Rust source code file that contains the module handlers for the file sink example.
+
```

Comment:
Remove that section, embedded those instructions with section that discuss about what Substreams changes are required.

Reply:
Removed.